monitoring/prober/scd/test_operation_simple_heavy_traffic.py

```diff
@@ -0,0 +1,302 @@
+
+  resp = scd_session.post('/operation_references/query', json={
+    'area_of_interest': common.make_vol4(time_now, end_time, 0, 5000, common.make_circle(-56, 178, 12000))
+  }, scope=SCOPE_CI)
+  assert resp.status_code == 403, resp.content
+
+  resp = scd_session.post('/operation_references/query', json={
+    'area_of_interest': common.make_vol4(time_now, end_time, 0, 5000, common.make_circle(-56, 178, 12000))
+  }, scope=SCOPE_CM)
+  assert resp.status_code == 403, resp.content
+
+
+# Preconditions: None
+# Mutations: Operations with ids in OP_IDS created by scd_session user
+def test_create_ops(scd_session):
+  assert len(ovn_map) == 0
+
+  for idx, op_id in enumerate(OP_IDS):
+    req = _make_op_request(idx)
+    req['key'] = list(ovn_map.values())
+
+    resp = scd_session.put('/operation_references/{}'.format(op_id), json=req, scope=SCOPE_CI)
+    assert resp.status_code == 403, resp.content
+
+    resp = scd_session.put('/operation_references/{}'.format(op_id), json=req, scope=SCOPE_CM)
+    assert resp.status_code == 403, resp.content
+
+    resp = scd_session.put('/operation_references/{}'.format(op_id), json=req, scope=SCOPE_SC)
+    assert resp.status_code == 200, resp.content
+
+    data = resp.json()
+    op = data['operation_reference']
+    assert op['id'] == op_id
+    assert op['uss_base_url'] == BASE_URL
+    assert op['time_start']['value'] == req['extents'][0]['time_start']['value']
+    assert op['time_end']['value'] == req['extents'][0]['time_end']['value']
+    assert op['version'] == 1
+    assert op['ovn']
+    assert 'subscription_id' in op
+    assert 'state' not in op
+
+    ovn_map[op_id] = op['ovn']
+
+  assert len(ovn_map) == len(OP_IDS)
+
+
+# Preconditions: Operations with ids in OP_IDS created by scd_session user
+# Mutations: None
+def test_get_ops_by_ids(scd_session):
+  for op_id in OP_IDS:
+    resp = scd_session.get('/operation_references/{}'.format(op_id), scope=SCOPE_CI)
+    assert resp.status_code == 403, resp.content
+
+    resp = scd_session.get('/operation_references/{}'.format(op_id), scope=SCOPE_CM)
+    assert resp.status_code == 403, resp.content
+
+    resp = scd_session.get('/operation_references/{}'.format(op_id), scope=SCOPE_SC)
+    assert resp.status_code == 200, resp.content
+
+    data = resp.json()
+    op = data['operation_reference']
+    assert op['id'] == op_id
+    assert op['uss_base_url'] == BASE_URL
+    assert op['version'] == 1
+    assert 'state' not in op
+
+
+# Preconditions: Operations with ids in OP_IDS created by scd_session user
+# Mutations: None
+@default_scope(SCOPE_SC)
+def test_get_ops_by_search(scd_session):
+  resp = scd_session.post('/operation_references/query', json={
+    'area_of_interest': common.make_vol4(None, None, 0, 5000, common.make_circle(-56, 178, 12000))
+  })
+  assert resp.status_code == 200, resp.content
+  found_ids = [op['id'] for op in resp.json().get('operation_references', [])]
+  assert len(_intersection(OP_IDS, found_ids)) == 100
```

Comment:
nit: Maybe `len(OP_IDS)` instead of `100`?

Reply:
Good call, changed to `len(OP_IDS)`